Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -5,7 +5,7 @@ import SwiftMarkup
 import SwiftSemantics
 import struct SwiftSemantics.Protocol
 
-#if os(Linux)
+#if !(os(iOS) || os(macOS) || os(tvOS) || os(watchOS))
```

Comment:
According to [this doc in swift-corelibs-foundation](https://github.com/apple/swift-corelibs-foundation/blob/main/Docs/ReleaseNotes_Swift5.md#dependency-management), the preferred way to check would be

```suggestion
#if canImport(FoundationNetworking)
```

Would that work for Windows?

Reply:
Yes, that should work for Windows. I didn't think of `canImport(FoundationNetworking)`.